app/src/main/java/ch/epfl/cs311/wanderwave/viewmodel/ProfileViewModel.kt

```diff
@@ -93,32 +115,117 @@ class ProfileViewModel @Inject constructor(private val repository: ProfileReposi
     _isInPublicMode.value = !_isInPublicMode.value
   }
 
-  fun fetchProfile(profile: Profile) {
-    // TODO : fetch profile from Spotify
-    // _profile.value = spotifyConnection.getProfile()....
-    // Fetch profile from Firestore if it doesn't exist, create it
-    Log.d("ProfileViewModel", "Fetching profile from Firestore...")
-    profileConnection.isUidExisting(profile.spotifyUid) { isExisting, fetchedProfile ->
-      if (isExisting) {
-        _profile.value = fetchedProfile ?: profile
-        // update profile on the local database
-        viewModelScope.launch {
-          val localProfile = repository.getProfile()
-          localProfile.collect { fetchedLocalProfile ->
-            if (fetchedLocalProfile != fetchedProfile) {
-              repository.delete()
-              repository.insert(fetchedProfile!!)
+  //  fun fetchProfile(profile: Profile) {
+  //    // TODO : fetch profile from Spotify
+  //    // _profile.value = spotifyConnection.getProfile()....
+  //    // Fetch profile from Firestore if it doesn't exist, create it
+  //    profileConnection.isUidExisting(profile.spotifyUid) { isExisting, fetchedProfile ->
+  //      if (isExisting) {
+  //        _profile.value = fetchedProfile ?: profile
+  //        // update profile on the local database
+  //        viewModelScope.launch {
+  //          val localProfile = repository.getProfile()
+  //          localProfile.collect { fetchedLocalProfile ->
+  //            if (fetchedLocalProfile != fetchedProfile) {
+  //              repository.delete()
+  //              repository.insert(fetchedProfile!!)
+  //            }
+  //          }
+  //        }
+  //      } else {
+  //        val newProfile = profile
+  //        profileConnection.addItem(newProfile)
+  //        viewModelScope.launch { repository.insert(newProfile) }
+  //        _profile.value = newProfile
+  //      }
+  //    }
+  //    // TODO : get rid of this line
+  //    profileConnection.getItem(profile.spotifyUid).let { Log.d("Firebase", it.toString()) }
+  //  }
+
```

Comment:
I think this comment should also be removed before merging into main.

Reply:
Was me sorry for that